.github/workflows/label.yml

```
@@ -6,7 +6,7 @@
 # https://github.com/actions/labeler
 
 name: Labeler
-on: [pull_request]
+on: [pull_request_target]
```

If we can use both triggers?
```suggestion
on: [pull_request, pull_request_target]
```